src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/LengthController.java

```diff
@@ -305,6 +305,9 @@ public Response lengthDensityGroupByTag(HttpServletRequest servletRequest,
           defaultValue = "", paramType = "query", dataType = "string", required = false),
       @ApiImplicitParam(name = "values2", value = ParameterDescriptions.DEPRECATED_USE_FILTER2,
           defaultValue = "", paramType = "query", dataType = "string", required = false),
+      @ApiImplicitParam(name = "filter", value = ParameterDescriptions.FILTER,
+          defaultValue = DefaultSwaggerParameters.HIGHWAY_FILTER2, paramType = "query",
```

Comment:
When just looking at the name of the constant, I would assume that `DefaultSwaggerParameters.HIGHWAY_FILTER2` is rather associated with the `filter2` parameter. The content though can of course be used for both filters. I would have connected them also fitting to the name, but we could also keep it like that.

Reply:
I have reversed the order here so that the resulting values are more easily readable as a percentage.